src/controller.rs

```diff
@@ -153,6 +159,18 @@ impl Controller for FoxBox {
         format!("ws://{}:{}/services/{}/", self.hostname, self.ws_port, service_id)
     }
 
+    fn get_config(&self, namespace: String, property: String, default: String)
+        -> String {
```

Comment:
I disagree. This feels inconsistent with the idea of the *default* parameter and it leads to very repetitive and ugly code in the callees. Been there.

The consistent alternative would be to rename this to *.get_config_with_default()*, and have *.get_config()* behave as you suggest.